about.html

```diff
@@ -0,0 +1,23 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
```

Comment:
Make sure to follow the same pattern in your title. I am seeing a different patter on each of the pages for the title. People will see these in the browser